pennylane/qnode_new/__init__.py

```diff
@@ -16,8 +16,8 @@
 
 .. currentmodule:: pennylane.qnode_new
 """
+from .base import BaseQNode, QuantumFunctionError
 from .cv import CVQNode
 from .decorator import qnode, QNode
 from .jacobian import JacobianQNode
```

Comment:
Is the intention here to rename the directory to ``qnode`` once the refactor has been completed entirely? If so, I think the naming is really nice, but otherwise, it could be confusing to see only ``.cv`` or ``.base``

Reply:
Yep, the intention is to rename to QNode :)